etcdserver/api/membership/member.go

```diff
@@ -51,6 +51,12 @@ type Member struct {
 	Attributes
 }
 
+// ConfigChangeContext represents a context for confChange.
+type ConfigChangeContext struct {
```

Comment:
Let's find a better place to define this. Conceptually it is not part of membership.member.
Maybe in server.go?

Reply:
better

Reply:
On second thought if we define it in etcdserver/server.go, it needs to import new pkg `go.etcd.io/etcd/etcdserver for `
ValidateConfigurationChange` to work. So import cycle will occur. I think maybe etcdserver/api/membership/cluster.go

Reply:
changed to etcdserver/api/membership/cluster.go ptla @jingyih